### DISCUSSION TITLE
Kinesis Video API (getMedia) support

### DISCUSSION THREAD

**Discussion by vbomfim** [etspaceman/kinesis-mock#486] [Q&A] [answered]
Does the mock support Kinesis Video API (getMedia)?

**Comment by etspaceman** [comment, marked as answer]
Not today. I could look into that but I'm not sure what the requirements would be here.

**Reply by vbomfim** [reply]
Thank you @etspaceman

**Reply by etspaceman** [reply]
This would be a pretty large undertaking and I don't think that I want to support this here. I could be convinced otherwise but this code base is already fairly massive and we would be looking at adding a _significant_ number of API calls, all of which I have no experience in leveraging.
